rel/files/ejabberd.cfg

```diff
@@ -473,11 +473,10 @@
 %%% ==================================
 %%% POOLS OF OUTGOING HTTP CONNECTIONS
 %%% ==================================
-%%
-%% {http_connections, [{conn1, [{server, "http://server.com:8080"},
-%%                              {pool_size, 50}]}
-%%                    ]}.
 
+{http_connections, [{http_pool, [{server, "http://localhost:8000"},
```

Comment:
Right, the default config should have no http pools enabled. Such a pool should be started in relevant test.